main/ertp/api/amount-math.md

```diff
@@ -4,29 +4,27 @@ Logic for manipulating `amounts`.
 
 ## Obtaining an AmountMath
 
-There are three ways and circumstances you can get access to an `amountMath`:
-- You made its associated `mint` and `issuer` with `makeIssuerKit()`, so use the `amountMath` returned from the call. 
-- You are writing a Zoe contract and the `issuer` is saved in Zoe, so call `zcf.getAmountMath(brand)`. 
-- You receive or learn about an `issuer` and are not writing a Zoe contract, so call `makeLocalAmountMath(issuer)`.
+To get access to an `amountMath`:
+- `import { amountMath } from '@agoric/ertp';`
+  - It's a record, not an object with a constructor.
```

Comment:
This is the wrong place for this comment. The `amountMath` that was just imported is an object with methods like `make()`, and `makeEmpty()`. Those both produce `amountMath` records that are not objects. That confusion is unfortunate.  Sorry.

Reply:
Removed record mention per our Keybase discussion.